### ISSUE TITLE
Passing variables via query params

### ISSUE THREAD

**Issue by duboff** [ForceDotComLabs/survey-force#31]
Hi, was wondering if you had a feature to pass a variable through a query param?

Example use case: we have an email with rating buttons 1 to 5. When the user clicks they get taken to a survey. We want to save the rating and ask them one follow up question.

**Comment by jrattanpal** [comment]
Hi,

can you provide more details? Currently it's not possible but I am looking to add "easy" features to this product.

If you can provide more details on the flow then I can look into this further

**Comment by duboff** [comment]
Certainly, this is very typical for Net Promoter Score or Review surveys. In them you ask a numerical question in the email and then a follow up question on the web survey. You use the score from the button to pre-fill the survey response to the numerical question. Each button in the email has a different URL (usually through a query param) and the survey uses that variable.

See this article from SurveyMonkey for example: https://www.surveymonkey.com/blog/2016/10/03/new-boost-completion-rates-embedded-questions-emails/

**Comment by agentile27** [comment]
Has anyone found a way to do this? I would love to be able to send an email with links for each choice 1-10 for the first question and have the appropriate choice preselected when clicking on the link
